consensus/src/epoch_manager.rs

```diff
@@ -839,6 +844,10 @@ impl EpochManager {
                     round = round_timeout_sender_rx.select_next_some() => {
                         monitor!("send_timeout", self.process_local_timeout(round));
                     },
+                    _ = progress_check_interval.select_next_some() => {
+                        info!("We're in the progress check interval branch!");
```

Comment:
too much to be a INFO?

Reply:
Every second? 🤷 Will hit debug if we have those logs 😄